presto-docs/src/main/sphinx/connector/hive-caching.rst

```diff
@@ -61,12 +61,16 @@ systems.
 Architecture
 ------------
 
-Caching for distributed storage provides a read-through cache. After first
-retrieval from storage by any query, objects are cached in the local cache
-storage on the workers. Objects are cached on local storage of each worker and
-managed by a BookKeeper component. Workers can request cached objects from other
-workers to avoid requests from the object storage. The cache chunks are 1MB in
-size and are well suited for ORC or Parquet format objects.
+Caching can operate in two modes. The default async mode provides the queried
+data directly and caches any objects asynchronously afterwards. Any following
+queries requesting the cached objects are served directly from the cache.
+
+The second mode is a read-through cache. After first retrieval from storage by
+any query, objects are cached in the local cache storage on the workers. Objects
+are cached on local storage of each worker and managed by a BookKeeper
+component. Workers can request cached objects from other workers to avoid
+requests from the object storage. The cache chunks are 1MB in size and are well
```

Comment:
`The cache chunks are 1MB in size and are well...` this is valid for both async and read-through mode

Reply:
ok .. I will separate the paragraph

Reply:
Separated the different concerns into paragraphs.